frontend/src/metabase/containers/ItemPicker/ItemPicker.unit.spec.js

```diff
@@ -138,7 +149,11 @@ async function setup({
     },
   );
 
-  await waitForElementToBeRemoved(() => screen.queryByText("Loading..."));
+  // sometimes it expects entities to be in the Redux store so there might not be a loading state
+  // ex: collections are stored in Redux, so there isn't a meaningful loading state here
+  if (waitForInitialLoad) {
```

Comment:
I'd drop this extra flag and made it work without id. Something like this could work:
```
await waitFor(() => {
  expect(screen.queryByText("Loading...")).not.toBeInTheDocument();
});
```

Reply:
Ok sweet I added that in, looks like the test still passes as well